Tests/GitHubContents.tests.ps1

```diff
@@ -0,0 +1,151 @@
+
+            $readmeFileBytes = Get-GitHubContent -OwnerName $script:ownerName -RepositoryName $repo.name -Path $readmeFileName -MediaType Html
+            $readmeFileString = [System.Text.Encoding]::UTF8.GetString($readmeFileBytes)
+
+            It "Should have the expected content" {
+                # Replace newlines with empty for comparison
+                $readmeFileString.Replace("`n", "").Replace("`r", "") | Should be $htmlOutput
+            }
+        }
+
+        Context 'For getting html (string) file contents' {
+
+            $readmeFileString = Get-GitHubContent -OwnerName $script:ownerName -RepositoryName $repo.name -Path $readmeFileName -MediaType Html -ResultAsString
+
+            It "Should have the expected content" {
+                # Replace newlines with empty for comparison
+                $readmeFileString.Replace("`n", "").Replace("`r", "") | Should be $htmlOutput
+            }
+        }
+
+        Context 'For getting object (default) file result' {
+
+            $readmeFileObject = Get-GitHubContent -OwnerName $script:ownerName -RepositoryName $repo.name -Path $readmeFileName
+
+            It "Should have the expected name" {
+                $readmeFileObject.name | Should be $readmeFileName
+            }
+            It "Should have the expected path" {
+                $readmeFileObject.path | Should be $readmeFileName
+            }
+            It "Should have the expected type" {
+                $readmeFileObject.type | Should be "file"
+            }
+            It "Should have the expected encoding" {
+                $readmeFileObject.encoding | Should be "base64"
+            }
+
+            It "Should have the expected content" {
+                # Convert from base64
+                $readmeFileString = [System.Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String($readmeFileObject.content))
+                $readmeFileString | Should be $rawOutput
+            }
+        }
+
+        Context 'For getting object file result as string' {
+
+            $readmeFileObject = Get-GitHubContent -OwnerName $script:ownerName -RepositoryName $repo.name -Path $readmeFileName -MediaType Object -ResultAsString
+
+            It "Should have the expected name" {
+                $readmeFileObject.name | Should be $readmeFileName
+            }
+            It "Should have the expected path" {
+                $readmeFileObject.path | Should be $readmeFileName
+            }
+            It "Should have the expected type" {
+                $readmeFileObject.type | Should be "file"
+            }
+            It "Should have the expected encoding" {
+                $readmeFileObject.encoding | Should be "base64"
+            }
+
+            It "Should have the expected content" {
+                $readmeFileObject.contentAsString | Should be $rawOutput
+            }
+        }
+
+        Remove-GitHubRepository -Uri $repo.svn_url
+    }
+}
+finally {
```

Comment:
formatting: Can you move this brace (and the one below in the `if`) to their own lines?  It's fine to leave the braces where they are for the `Context`/`Describe`/`It` expressions.